NotationV0/src/install.ts

```diff
@@ -16,11 +16,8 @@ export async function install(): Promise<void> {
         downloadURL = taskLib.getInput('url', true) || '';
         checksum = taskLib.getInput('checksum', true) || '';
     } else {
-        const versionRange = taskLib.getInput('version', true);
-        if (!versionRange) {
-            throw new Error('Version is not specified');
-        }
-        const downloadInfo = getDownloadInfo(versionRange, NOTATION_VERSION_FILE);
+        const versionPrefix = taskLib.getInput('version', true) || '';
```

Comment:
`version` is required field, and empty string is also not valid, so don't need to check it again. Just convert it from `string | undefined` to be `string`